index.js

```diff
@@ -95,6 +95,13 @@ var createResultTable = function (caption) {
   };
 };
 
+var sectionCheck = function (options, key) {
+  return options.sections.some(function (section) {
+    var length = section.length;
+    return length > 2 && key.slice(0, length) === section;
```

Comment:
Rather than an implicit relationship, let's please use an explicit map of option → section name.